mantidproject/mantid
# Clean up unused CMake config

**Description of work.**
Removal of a lot of old CMake configuration, including:
- old CPack and packaging scripts
- old third party Windows setup
- pre-Conda dependency package configuration
- old Conda recipe config
- a lot of other things that are not being used any more


**To test:**
Check that the following pipeline build succeeds in building all packagages:
https://builds.mantidproject.org/job/build_packages_from_branch/244/

Fixes #35074

*This does not require release notes* because **it is a tidy up of our build config.**



---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
